snuba/datasets/processors/replays_processor.py

```diff
@@ -274,6 +271,51 @@ def _encode_utf8(value: str) -> str:
     return value.encode("utf8", errors="backslashreplace").decode("utf8")
 
 
+def datetimeify(value: Any) -> datetime:
+    """Return a datetime instance or err.
+
+    Datetimes for the replays schema standardize on 32 bit dates.
+    """
+    return _timestamp_to_datetime(_collapse_or_err(_collapse_uint32, _intify(value)))
+
+
+def _intify(value: Any) -> int:
+    """Return an integer or err."""
```

Comment:
Doesn't the `int` cast already do that? It will throw a `ValueError` if the thing it is casting is not int-able.

Reply:
Ah, duh..  replaced with `int`.